target_chains/fuel/contracts/pyth-interface/src/data_structures/wormhole_light.sw

```diff
@@ -115,8 +115,20 @@ impl GuardianSetUpgrade {
         while i < guardian_length {
             let (_, slice) = encoded_upgrade.split_at(index);
             let (key, _) = slice.split_at(20);
-            let key: b256 = b256::from_be_bytes(key.clone());
-            new_guardian_set.keys.push(key.rsh(96));
+            let mut full_address_key = Bytes::with_capacity(32);
+
+            // Sway's from_be_bytes expects a 32-byte array, so we pad with zeros
```

Comment:
looks like this is repeated in several places, maybe a helper function would be good?